data/brands/shop/convenience.json

```diff
@@ -63,6 +63,17 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "Smart Point",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["Reliance Smart","Reliance Smart Point","Jio Smart","Jio Smart Point"],
+      "tags": {
+        "brand": "Reliance Smart Point",
+        "brand:wikidata": "Q7311028",
```

Comment:
Probably should have a separate brand item instead of just putting everything under Reliance Retail.

Reply:
```suggestion
        "brand:wikidata": "Q123565801",
```